Cargo.toml

```diff
@@ -13,7 +13,7 @@ dlpi = { git = "https://github.com/oxidecomputer/dlpi-sys", branch = "main" }
 indicatif = "0.17.2"
 libc = "0.2.137"
 libloading = "0.7.3"
-p4rs = { git = "https://github.com/oxidecomputer/p4", branch = "main" }
+p4rs = { git = "https://github.com/oxidecomputer/p4", rev = "5d84413c11c745a8debbd70cd05f2c63ae9a7f8d" }
```

Comment:
Let's pin this on the latest rev `d59491f86668e15e92f35ffe3f94504ff4241c64`

Reply:
Done.